### DISCUSSION TITLE
How to remove "Draggable" from entire area so that only pieces can move?

### DISCUSSION THREAD

**Discussion by nirvitarka** [flbulgarelli/headbreaker#54] [Q&A] [answered]
How to remove "Draggable" from entire area so that only pieces can move and not the entire puzzle

**Comment by flbulgarelli** [comment, marked as answer]
👋 Hi @nirvitarka 

Have you checked [this example](https://flbulgarelli.github.io/headbreaker/#offstage-drag-prevention)? You can use the `fixed: true` config to prevent motion of the canvas, while still allowing pieces to be dragged.

Hope that helps!
